google-cloud-translate/test/helper.rb

```diff
@@ -48,6 +49,8 @@ def stub_response(status, body)
   end
 
   def post(*_)
+    @last_request = OpenStruct.new headers: {}, body: ""
+    yield @last_request if block_given?
```

Comment:
not sure why the yield is necessary

Reply:
The usage of the class passes a block that modifies the request object (to add headers, etc.)